scripts/workspace_status.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+"""Bazel `--workspace_status_command` script.
+
+This script is designed to be used in Bazel`s `--workspace_status_command` and
+generate any desirable status artifacts.
+"""
+
+__copyright__ = """
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""
+
+import subprocess
+
+
+def git_commit_sha() -> str:
+    return subprocess.check_output(
+        ["git", "rev-parse", "--short", "HEAD"], shell=False, text=True
+    ).strip()
+
+
+def git_dirty_suffix() -> str:
+    status = subprocess.check_output(
+        ["git", "status", "--porcelain"], shell=False, text=True
+    ).strip()
+    return ".dirty" if len(status) > 0 else ""
```

Comment:
Please write this out as a full if/else. Maybe this is my familiarity with C++ speaking, but I think Python's ternaries are difficult to read.

Reply:
Sure, I was just trying to use the language features. I somewhat agree with you about it being hard to read.